### PR TITLE
Fix/modal small variation

### PR DESCRIPTION
**Pull request by clayton-duarte**
Fixes Modal small variation style and behaviour.

Contextual screenshot after fix:

screenshot-localhost_8611-2020 12 08-17_19_15

### CODE REVIEW

File: src/components/Modal/index.tsx [hunk 1]
```diff
@@ -39,9 +41,9 @@ const StyledModal = styled(Grid)<ModalProps>`
   overflow-y: auto;
   @media (max-width: ${props => props.theme.breakpoints.SM}) {
     border-radius: ${props => (props.small ? props.theme.shape.RADIUS : 0)};
+    max-width: calc(100% - ${props => (props.small ? '48px' : '0px')});
```
Comment (mitchied) [inline src/components/Modal/index.tsx:44]: `48px` that is so specific ..what we basing this on?

Comment (clayton-duarte) [reply src/components/Modal/index.tsx:44]: hehe, the base size is 24px, I was thinking about writing `calc(100% - 2 * 24px)` but I think people will ask the same thing... BTW, that reminds me we need to add a `theme.shape.SPACING = "24px"` property to the theme

Comment (clayton-duarte) [reply src/components/Modal/index.tsx:44]: At least that's the base spacing on OOF, VAF might have a different one, but if it's part of the theme so we can easily use different values

Comment (mitchied) [reply src/components/Modal/index.tsx:44]: yess it's much helpful when trying to figure out why it was set to a specific value. We def need to be defining base variables

**Review by pajanpl — APPROVED** [review]

**Review by mitchied — APPROVED** [review]

**Pull request merged**